src/rcl_lifecycle_bindings.cpp

```diff
@@ -53,6 +53,7 @@ static v8::Local<v8::Object> wrapTransition(
 NAN_METHOD(CreateLifecycleStateMachine) {
   RclHandle* node_handle = RclHandle::Unwrap<RclHandle>(
       Nan::To<v8::Object>(info[0]).ToLocalChecked());
+  bool enableCommunicationInterface = Nan::To<bool>(info[1]).FromJust();
```

Comment:
nit: enableCommunicationInterface => enable_communication_interface

Reply:
I renamed the var to `enable_com_interface` which is the same name as the options property.